extensions/azure/processors/PutAzureBlobStorage.cpp

```diff
@@ -22,41 +22,14 @@
 
 #include "core/ProcessContext.h"
 #include "core/ProcessSession.h"
+#include "core/PropertyBuilder.h"
```

Comment:
I left these in on purpose, as I thought I'd move the static definitions back from the `...StaticDefinitions.cpp` soon.  Since it looks like they will stay like that for a while, I have removed these includes in 416f85452eb9a640de42bc52c9619d459bc5e5cf